tweepy/models.py

```diff
@@ -231,6 +231,7 @@ class DirectMessage(Model):
     @classmethod
     def parse(cls, api, json):
         dm = cls(api)
+        setattr(dm, '_json', json)
         if "event" in json:
             json = json["event"]
```

Comment:
This should probably be after `json` is set to the specific event that [GET direct_messages/events/show](https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/get-event) (`API.get_direct_messsage`) returns, so that the JSON represents the actual event and so as to be consistent with the JSON that would be representing a Direct Message event when returned by [GET direct_messages/events/list](https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/list-events) (`API.list_direct_messages`):
```suggestion
        if "event" in json:
            json = json["event"]
        setattr(dm, '_json', json)
```